ui/main/src/app/business/templateGateway/userCardTemplateGateway.ts

```
@@ -0,0 +1,240 @@
+    public static setStartDate(value: number) {
+        UserCardTemplateGateway._startDate = value;
+    }
+
+    public static setTemplateListenerForEntityUsedForSendingCard(listener: Function) {
+        UserCardTemplateGateway._functionToSendEntityUsedForSendingCardToTemplate = listener;
+    }
+
+    public static setUserEntityChildCard(value: any) {
+        UserCardTemplateGateway._userEntityChildCard = value;
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)